client/src/components/participant-form/index.js

```diff
@@ -35,15 +35,23 @@ export const Form = ({
   const formValues = initialValues
     ? initialValues
     : {
+        program: '',
         eligibility: '',
+        educationalRequirements: '',
         firstName: '',
         lastName: '',
         phoneNumber: '',
         emailAddress: '',
         postalCode: '',
+        indigenous: '',
         preferredLocation: [],
+        driverLicense: '',
+        experienceWithMentalHealthOrSubstanceUse: '',
         consent: '',
         reasonForFindingOut: [],
+        currentOrMostRecentIndustry: '',
+        roleInvolvesMentalHealthOrSubstanceUse: '',
+        otherIndustry: '',
```

Comment:
Isn't it possible to merge this into currentOrMostRecentIndustry?

Reply:
I do have code to do this saved locally, but I wanted to get some opinions! (also this other option may even be removed)

my thought  was having both would make it more apparent of the users `Other` industry as opposed to just having the specified value in `currentOrMostRecentIndustry`.  That way we know the user had selected Other + there industry.

Probably a stretch scenario, but if we ever wanted to sort by industry for example

Reply:
There might be pros and cons, but the mismatch of questions and fields might cause more trouble than we can expect now.

Reply:
I agree with Sunghwan that I don't see a huge value add of putting this in a different field